install.sh

```diff
@@ -6,15 +6,16 @@ if [[ -n "$MSYSTEM" ]]; then
   exit 1
 fi
 
+# Thanks to https://unix.stackexchange.com/a/145654/108960
+log_file="sentry_install_log-`date +'%Y-%m-%d_%H-%M-%S'`.txt"
+exec &> >(tee -a "$log_file")
+
+echo "::group::Defining variables and helpers ..."
```

Comment:
This is nice in GitHub Actions but it will clobber our logs and other people's terminals during install. Maybe detect GHA and limit this there?

Reply:
Done in de6bcf018ab6e3fab0ea5ff837c74d3d77c804fe for `install.sh`. Leaving `test.sh` groups as-is since that is CI-centric.